src/GraphqlClient/HTTP.purs

```diff
@@ -133,25 +135,48 @@ tryDecodeGraphqlResponse decoderForData jsonBody = graphqlResponseOrError # (\er
           (possiblyParsedData :: PossiblyParsedData parsed) <- decoderForData dataJson # (\error -> Right $ UnparsedData error dataJson) \/ (Right <<< ParsedData)
           Right $ Left $ GraphqlUserError errors possiblyParsedData
 
-post :: Affjax.URL -> Array Affjax.RequestHeader -> ArgonautCore.Json -> Aff (Either Affjax.Error (Affjax.Response ArgonautCore.Json))
-post url headers body =
-  Affjax.request (Affjax.defaultRequest
-                 { method = Left POST
-                 , url = url
-                 , content = Just $ Affjax.RequestBody.json $ ArgonautCodecs.Encode.encodeJson body
-                 , responseFormat = Affjax.ResponseFormat.json
-                 , headers = headers
-                 })
+type Request = Affjax.Request ArgonautCore.Json
```

Comment:
I once wanted to make an alias `type Decoder a = Json -> Either String a`

and was told that it's bad

https://github.com/purescript-contrib/purescript-argonaut-codecs/pull/74#discussion_r431510193

I dont do aliases like this anymore

Reply:
removed